tutorials/Native_distributions_and_local_execution/README.md

```diff
@@ -408,3 +409,79 @@ compose.desktop {
     }
 }
 ```
+
+## Customizing Info.plist on macOS
+
+We aim to support important platform-specific customization use-cases via declarative DSL.
+However, the provided DSL is not enough sometimes. If you need to specify `Info.plist`
+values, that are not modeled in the DSL, you can work around by specifying a piece
+of raw XML, that will be appended to the application's `Info.plist`.
+
+### Example: deep linking into macOS apps
+
+1. Specify a custom URL scheme:
+``` kotlin
+// build.gradle.kts
+compose.desktop {
+    application {
+        mainClass = "MainKt"
+        nativeDistributions {
+            targetFormats(TargetFormat.Dmg)
+            packageName = "Deep Linking Example App"
+            macOS {
+                bundleID = "org.jetbrains.compose.examples.deeplinking"
+                infoPlist {
+                    extraKeysRawXml = macExtraPlistKeys
+                }
+            }
+        }
+    }
+}
+
+val macExtraPlistKeys: String
+    get() = """
+      <key>CFBundleURLTypes</key>
```

Comment:
Docs at https://docs.oracle.com/en/java/javase/12/docs/api/java.desktop/java/awt/Desktop.html#setOpenURIHandler(java.awt.desktop.OpenURIHandler) says that it only works when `CFBundleDocumentTypes` is present. Is it actually correct?

Reply:
Does not seem so (I've checked the example on Big Sur and it worked)